console/src/main/java/org/georchestra/console/ws/backoffice/users/UsersExport.java

```diff
@@ -68,6 +75,15 @@
     @RequestMapping(method = RequestMethod.GET, value = "/account/gdpr/download", produces = "application/zip")
     public void downloadUserData(HttpServletResponse response)
             throws NameNotFoundException, DataServiceException, IOException {
+        /*
+         * Disabling this endpoint is the gdpr.enable property is set to false. we are
+         * not using spring boot, so we cannot disable this endpoint with
+         * the @ConditionalOnProperty annotation.
```

Comment:
not sure it is necessary to mention the fact that the annotation is unavailable

Reply:
Yeah, dunno, I was just bored that I found this perfect annotation in the doc and then my IDE said "Nope, you can't use that" So I mentioned it, but you're probably right.